scripts/lib/api/processHtml.ts

```diff
@@ -279,24 +279,23 @@ export function processMembersAndSetMeta(
     }
 
     const $dl = $(dl);
+    const id = $dl.find("dt").attr("id") || "";
```

Comment:
Excellent commit. This is really helpful to better understand what is child-specific vs from the parent.